cmd/main.go

```diff
@@ -224,7 +239,12 @@ func runDev(ctx context.Context, args []string) error {
 	}
 
 	if os.Getenv("XCADDY_SETCAP") == "1" {
```

Comment:
These lines seem to 100% match the lines `runBuild`. Is it possible to dedup this?